src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -428,39 +418,47 @@ const ActionLog = () => {
                     </TableRow>
                 </TableHead>
                 <TableBody>
-                    {data.logs.results.map((actionItem) => {
-                        return (
-                            <TableRow>
-                                <TableCell>
-                                    <Moment
-                                        date={actionItem.date}
-                                        format="dddd, M/D/YYYY h:mm a"
-                                    />
-                                </TableCell>
-                                <TableCell>
-                                    {
-                                        renderName(actionItem.userId)
-                                    }
-                                </TableCell>
-                                <TableCell>
-                                    <div style={{ textTransform: "capitalize" }}>
-                                        {actionItem.adminType.toLowerCase()}
-                                    </div>
-                                </TableCell>
-                                <TableCell>
-                                    {renderChip(actionItem.action)}
-                                </TableCell>
-                                <TableCell>
-                                    <div style={{ textTransform: "capitalize" }}>
-                                        {actionItem.objectType}
-                                    </div>
-                                </TableCell>
-                                <TableCell>
-                                    {actionItem.objectRepr}
+                    {data.logs.results.length == 0
+                        ? (<TableRow>
+                            <TableCell> 
+                                {//ASK DESIGN FOR STYLING ON THIS
+                                }
+                                There was no match found for your data
                                 </TableCell>
-                            </TableRow>
-                        )
-                    })}
+                        </TableRow>)
+                        : data.logs.results.map((actionItem) => {
+                            return (
```

Comment:
you should use an [implicit return](https://jaketrent.com/post/javascript-arrow-function-return-rules/#:~:text=For%20an%20arrow%20function%20to,code%20without%20a%20resulting%20value.) here instead of `.map((actionItem) => { return(...) }`.
example:
```
.map( actionItem => (<TableRow>
         ...rest of code
</TableRow>))
```